lib/internal/linkedlist.js

```diff
@@ -6,6 +6,13 @@ function init(list) {
 }
 exports.init = init;
 
+// create a new linked list
+function create() {
+  var list = { _idleNext: null, _idlePrev: null };
```

Comment:
I also wonder if there would be any performance benefit to instead creating a new instance of an object that doesn't inherit from `Object.prototype`? For example:

```js
function LinkedListNode() {
  this._idleNext = null;
  this._idlePrev = null;
}
LinkedListNode.prototype = Object.create(null);

function create() {
  const list = new LinkedListNode();
  init(list);
  return list;
}
```